paimon-core/pom.xml

```diff
@@ -161,6 +161,14 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
It is better to not introduce mockito. Mockito is not good test framework, we should not introduce it until it is absolutely necessary, we introduce our own abstraction.